src/Web/WebClient.php

```diff
@@ -287,6 +287,14 @@ protected function detectBrowser($userAgent)
             return;
         }
 
+        // Check for Google's Private Prefetch Proxy
+        if ($userAgent === 'Chrome Privacy Preserving Prefetch Proxy') {
+            // Private Prefetch Proxy does not provide any further details like e.g. version
+            $this->browser  = self::CHROME;
+
+            return;
+        }
+
         $patternBrowser = '';
```

Comment:
```suggestion
```
This is assigned when there is match otherwise it is an unnecessary assignment.

Reply:
It won't reach/execute line 333 without `$this->browser`. Only with a match that `$this->browser` and `$patternBrowser` will be assigned. I understand your choice. It can be done in a separate PR.

Reply:
Yes, you are right. But I prefer a separate PR.